robottelo/ui/oscappolicy.py

```diff
@@ -27,6 +27,8 @@ def create(self, name, desc=None, content=None, profile=None, period=None,
         if desc:
             self.text_field_update(locators['oscap.desc_policy'], desc)
         self.click(common_locators['submit'])
+        if self.wait_until_element(common_locators['haserror']):
+            return
```

Comment:
This is unique case, where in we move from one form to another ( verification of the form happens one by one) and only in the end that it submits. So there is no way I can think of currently which can be done, as we used to do earlier or for others.

What this does is, it returns if the `general` info form has any issues and asserts that the entity is indeed None ( not created due to the negative scenarios) .